### DISCUSSION TITLE
Why NvChad startup so fast?

### DISCUSSION THREAD

**Discussion by Song-Tianxiang** [NvChad/NvChad#2675] [Q&A] [answered]
Does there exist an article to explain what happened? I have noticed some `bytecode` for base46&ui plugin and a custom lazy load function `core.utils.lazy_load`. Basically, what makes NvChad start so fast? How can I achieve the same startup time for my own neovim config without NvChad.

**Comment by siduck** [comment, marked as answer]
@Song-Tianxiang that lazy loadfunction is replaced with an autocmd now btw,  look at each plugin that nvchad lazyloads
